learn/tanssi/technical.md

```diff
@@ -1,53 +1,95 @@
 ---
-title: Technical Features
-description: Tanssi is an Appchain protocol that eases the process of deploying Appchains so that developers can focus on their custom logic.
+title: Technical Features of Tanssi
+description: Tanssi provides block production services assigning collators to the ContainerChains, requiring minimal changes to the code for Appchains to be deployed.
 ---
 
-# What is Tanssi? {: #what-is-tanssi } 
+# Technical Features of Tanssi {: #technical-features-of-tanssi } 
 
-Tanssi is a protocol that makes deploying blockchains with custom logic specific to certain types of applications a breeze. These customized blockchains are normally referred to as Appchains, but in Tanssi terms, they are also known as Container Chains.
+## Introduction {: #introduction } 
 
-## The Problem
+As presented in the [Overview](/learn/tanssi/overview){target=_blank} article, Tanssi is an Appchain infrastructure protocol that streamlines the deployment of blockchains with custom logic specific to a wide range of use cases, including DeFi, NFTs, Gaming, and any other use case development teams may want to address.
 
-Appchains normally have to deal with the following problems:
+Infrastructure poses a huge challenge for developers who would need to bootstrap collators for block production, data preservers, RPC endpoints, and deal with integrations and interoperability, assigning precious effort and resources and losing focus on what is really important: the Appchain Runtime, the UX, and the value proposition to the users.
 
-- **Complex Infrastructure Management** - Appchain deployments typically demand handling numerous infrastructural components like block producers, validators, wallets, block explorers, indexers, RPC endpoints, and more. This is both time and resource consuming
+In Tanssi terms, Appchains are called ContainerChains, similar to the concept coined in [Docker](https://www.docker.com){target=_blank}, allowing teams to focus on the product while alleviating deployment-related issues. In this analogy, the Tanssi network resembles [Kubernetes](https://kubernetes.io){target=_blank}, in its role as orchestrator, managing resources to guarantee the liveness and performance of the ContainerChains.
 
-- **Weak & Inefficient Security** - Appchains commonly suffer from having small or weak validator sets. Early-stage Appchains don't have enough economic guarantees to power a robust consensus mechanism. Moreover, paying for full blockchain capacity is inefficient when only a fraction of it is needed
+In this article, we discuss how to build and deploy your own Modular Blockchain and the following technical aspects of the Tanssi protocol:
```

Comment:
We are not stating how to build and deploy your own Modular Blockchain right? Just explaining the technical aspects of Tanssi in more depth